packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -0,0 +1,213 @@
+          </td>
+        </tr>
+        <tr v-if="highlightedFile.ownerDisplayName" data-test-id="ownerName">
+          <th class="oc-pr-s" v-text="ownerTitle" />
+          <td>
+            <p class="oc-m-rm">
+              {{ highlightedFile.ownerDisplayName }}
+              <span v-if="isCurrentUser" v-translate>(me)</span>
+            </p>
+          </td>
+        </tr>
+        <tr data-test-id="sizeInfo">
+          <th class="oc-pr-s" v-text="sizeTitle" />
+          <td v-text="getResourceSize(highlightedFile.size)" />
+        </tr>
+        <tr v-if="versions.length > 0 && !isPublicPage" data-test-id="versionsInfo">
+          <th class="oc-pr-s" v-text="versionsTitle" />
+          <td>
+            <oc-button
+              v-oc-tooltip="seeVersionsLabel"
+              appearance="raw"
+              :aria-label="seeVersionsLabel"
+              @click="expandVersionsAccordion()"
+              v-text="versions.length"
+            />
+          </td>
+        </tr>
+      </table>
+    </div>
+  </div>
+</template>
+<script>
+import Mixins from '../../../mixins'
+import MixinResources from '../../../mixins/resources'
+import MixinRoutes from '../../../mixins/routes'
+import { mapActions, mapGetters, mapMutations } from 'vuex'
+import { ImageDimension, ImageType } from '../../../constants'
+
+export default {
+  mixins: [Mixins, MixinResources, MixinRoutes],
+  title: $gettext => {
+    return $gettext('Details')
+  },
+  data: () => ({
+    loading: false
+  }),
+  computed: {
+    ...mapGetters('Files', ['highlightedFile', 'versions']),
+    ...mapGetters(['getToken']),
+    ...mapGetters(['user']),
+
+    detailSharingInformation() {
+      return this.$gettext(
+        'This file has been shared. Manage access rights by clicking on the icons on the left or on the sections below'
+      )
+    },
+    detailsTableLabel() {
+      return this.$gettext('Overview on the information about the selected file')
+    },
+    sharesLabel() {
+      return this.$gettext('Show invited people')
+    },
+    linksLabel() {
+      return this.$gettext('Show links')
+    },
+    lastModifiedTitle() {
+      return this.$gettext('Last modified:')
+    },
+    ownerTitle() {
+      return this.$gettext('Owner:')
+    },
+    sizeTitle() {
+      return this.$gettext('Size:')
+    },
+    versionsTitle() {
+      return this.$gettext('Versions:')
+    },
+    seeVersionsLabel() {
+      return this.$gettext('See all versions')
+    },
+    capitalizedModifiedDate() {
+      const lastModifiedDate = this.formDateFromNow(this.highlightedFile.mdate, 'Http')
+      return lastModifiedDate.charAt(0).toUpperCase() + lastModifiedDate.slice(1)
+    },
+    hasAnyShares() {
+      return (
+        this.highlightedFile.shareTypes.length > 0 || this.highlightedFile.indicators.length > 0
+      )
+    },
+    hasPeopleShares() {
+      return (
+        this.highlightedFile.shareTypes.includes(0) ||
```

Comment:
Please use the `shareTypes` constants.

Reply:
Also, checking for shareType `0` is not sufficient, is it? The people shares accordion shows other share types as well.